### PR TITLE
added _acceleration to Resampler update method

### PR DESCRIPTION
**Pull request by OkonSamuel**
- added _acceleration parameter to `update` method of `Resampler` (Important omission)
- removed redundant `@sync` and `close` methods. (Turns out the lock in `close` method also contributed to the issue)
- Better workaround to the silent warnings issue for the `evaluate` method.
cc @ablaom

### CODE REVIEW

File: src/resampling.jl [hunk 1]
```diff
@@ -925,11 +931,14 @@ function MLJBase.update(resampler::Resampler{Holdout},
         _process_weights_measures(resampler.weights, resampler.measure,
                                   mach, resampler.operation,
                                   verbosity, resampler.check_measure)
+
+    _acceleration = _process_accel_settings(resampler.acceleration)
+
```
Comment (ablaom) [inline src/resampling.jl:936]: Good catch

**Comment by ablaom** [comment]
Many thanks. Merge when ready.

**Comment by OkonSamuel** [comment]
@ablaom should i tag a new release. Or should would you mind doing that??

**Comment by ablaom** [comment]
Oops. This was merged directly to master and not through dev. 

@OkonSamuel Would be great if you could update dev and tag a new release. Otherwise, I will do it on Monday.

**Comment by OkonSamuel** [comment]
@ablaom. I tagged a new release earlier. 
At that time the **master** and **dev** were in sync so there isn't any need to update the **dev**. (the **dev** is just a commit ahead of **master**)

**Pull request merged**